src/HealthChecks.Elasticsearch/DependencyInjection/ElasticsearchHealthCheckBuilderExtensions.cs

```diff
@@ -24,10 +24,14 @@ public static IHealthChecksBuilder AddElasticsearch(this IHealthChecksBuilder bu
         {
             var options = new ElasticsearchOptions();
             options.UseServer(elasticsearchUri);
-            
+
+            IHealthCheck healthCheck = options.CheckClusterHealth ?
```

Comment:
I prefer create two different overloads for IHealthCheckBuilders extensions, probably with different options ( for clustering and single ES), in this case this code is invalid